packages/components/ScreenContainer/index.tsx

```diff
@@ -67,6 +69,14 @@ export const ScreenContainer: React.FC<{
   forceNetworkKind,
   forceNetworkFeatures,
 }) => {
+  const dispatch = useAppDispatch();
```

Comment:
can we move this to a component in App.tsx?
(see WalletSyncer component)

Reply:
done; thanks for the feedback